### ISSUE TITLE
jenkins : Did not find any matching files. Setting build result to FAILURE.

### ISSUE THREAD

**Issue by JasonFengJ9** [eclipse/openj9#8378]
Failure link
------------

Optional info
-------------

Failure output (captured from console output)
---------------------------------------------

```
Test_openjdk8_j9_extended.functional_x86-64_linux_Nightly/269
23:05:27  TAP Reports Processing: START
23:05:27  Looking for TAP results report in workspace using pattern: **/*.tap
23:05:27  Did not find any matching files. Setting build result to FAILURE.
```

**Comment by smlambert** [comment]
This issue does not link to any real example.  While I can search for the OpenJ9 jobs called by this name, it makes it easier to triage if you include links to real Jenkins jobs.  If its a report about internal jobs, you can still state that (even though you will not include internal links)

**Comment by JasonFengJ9** [comment]
Yeah, this is from internal builds. I was hoping `Test_openjdk8_j9_extended.functional_x86-64_linux_Nightly/269` provides the hints to the actual links.

**Comment by smlambert** [comment]
Got it now, thanks!
